guides/release/code-editors/index.md

```diff
@@ -0,0 +1,61 @@
+During development, add Ember.js specifc extensions to your code editor for added functionality. Below is a list of some of the extensions available:  
```

Comment:
```suggestion
During development, add Ember.js specific extensions to your code editor for added functionality. Below is a list of some of the extensions available, many of which are created and maintained by the developer community:  
```
Just adding a little bit of messaging to give credit to the authors and to avoid the assumptions that the core team maintains them.